i18n/en.yaml

```diff
@@ -523,6 +526,7 @@ FALLOUT.UI.SPECIAL_ABILITY: Special Ability
 FALLOUT.UI.stashed: Stashed
 FALLOUT.UI.Type: Type
 FALLOUT.UI.UNSORTED: Unsorted
+FALLOUT.UI.USINGLUCK: Using luck
```

Comment:
Similarly to comment above, change this to `FALLOUT.UI.using_luck`

Reply:
done